.github/workflows/update-acl.yml

```diff
@@ -0,0 +1,127 @@
+name: update-acl
+
+on:
+  issues:
+    types:
+      - labeled
+
+jobs:
+  check_permission:
+    if: github.event.label.name == 'approved'
+    runs-on: ubuntu-latest
+    steps:
+      - name: check privilege
+        run: |
+          maintainers=(
+            "joeaba"
```

Comment:
why just me?

Reply:
Ideally the update acl pipeline will be triggered by the on board pipeline. I think we will use your PAT in the on board pipeline so I only write your name here. If we would like to make this step can be triggered by others manually, we can have more names here!

Reply:
yeah ok that's fine!